index.js

```diff
@@ -1,16 +1,64 @@
+/* jshint node: true */
+/* jshint asi: true */
 var connect = require('connect')
 
 var config = require('./config.json')
 
-function helmet () {
+function findModuleName(alias) {
+  'use strict';
+  var result = null;
+  Object.keys(config.middlewares).forEach(function (moduleName) {
+    if (alias === moduleName || config.middlewares[moduleName].indexOf(alias) > -1) {
+      result = moduleName
+    }
+  })
+  return result
+}
+
+function helmet (optionOverrides) {
+  'use strict';
   var chain = connect()
+
+  // init options for default middleware
+  var options = {}
   config.defaultMiddleware.forEach(function (middlewareName) {
-    chain.use(helmet[middlewareName]())
+    var moduleName = findModuleName(middlewareName)
+    // TODO: initialize with real default props from module?
+    options[moduleName] = {}
```

Comment:
Looks like none of the modules have predefined options, only predefined behavior. The TODO flag can be dropped..